api/agent/drivers/docker/cookie.go

```diff
@@ -16,7 +16,15 @@ import (
 	"go.opencensus.io/trace"
 )
 
-var ErrImageWithVolume = models.NewAPIError(http.StatusBadRequest, errors.New("image has Volume definition"))
+const (
```

Comment:
Two thoughts here:

- Seems like this would benefit from a comment that explains the interdependence between the uid/gid and the runtime in the container.  IOW, if somebody comes along and changes this code for their own purpose, it would be nice to give them a hint that they also need to potentially change what's going on inside their container.
- Is it worthwhile to set this as a default to 1000 but then allow users to override?  I'm fine with the answer that this is a "patches welcome" case if somebody else wants it, but figured it would be worth asking up front.